packages/core/src/api/index.ts

```diff
@@ -1,7 +1,3 @@
 export * from './services';
 export * from './apolloClient';
-export {
-  ContentNodeOptions,
-  ListPostOptions,
-  categoryOptions,
-} from './queries';
+export * from './queries';
```

Comment:
Quite a few imports from [`packages/react/src/hooks.ts`](https://github.com/wpengine/headless-framework/pull/217/files#diff-c6ce8673cb12abb4ff3134573c3d9a414e9063997fbc8cddf6912cd8bcac3ff5R4) relies on `queries` that were not previously exported.